transports/janus_mqtt.c

```diff
@@ -154,18 +158,32 @@ typedef struct janus_mqtt_context {
 	char *cert_file;
 	char *key_file;
 	gboolean verify_peer;
+#ifdef MQTTVERSION_5
+	int vacuum_interval;
+#endif
 } janus_mqtt_context;
 
+#ifdef MQTTVERSION_5
+/* MQTT 5 specific types */
+typedef struct janus_mqtt_transaction_state {
+	MQTTProperties *properties;
+	time_t created_at;
```

Comment:
Why the `time_t`? It may be easier (and probably more easily compatible on other systems) to use a `gint64` here and rely on `janus_get_monotonic_time()` instead.